bundle/Templating/Twig/Extension/QueryExtension.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace Netgen\Bundle\EzPlatformSiteApiBundle\Templating\Twig\Extension;
+
+use Netgen\Bundle\EzPlatformSiteApiBundle\QueryType\QueryExecutor;
+use Netgen\Bundle\EzPlatformSiteApiBundle\View\ContentView;
+use Twig_Error_Runtime;
+use Twig_Extension;
+use Twig_SimpleFunction;
+
+/**
+ * Twig extension for executing queries from the QueryCollection injected
+ * into the template.
+ */
+class QueryExtension extends Twig_Extension
```

Comment:
What's the usecase for running queries from templates?

Reply:
I'm really looking for a better syntax when accessing queries from template. Calling a function seems more visible/explicit then accessing a variable. Note that this still can work only the QueryCollection that was injected to the template, so it's not for running arbitrary queries.

In any case queries need to be executed from template, unless we pre-execute them and inject the results instead.

Reply:
My question was about the second point. I'd rather see that queries were executed from an event listener and results injected.

Reply:
OK, discussed this f2f - it's mostly a matter of preference, let's see what @MarioBlazek has to say :)